### PR TITLE
Update some cargo deps

### PR DESCRIPTION
**Pull request by Kailai-Wang**
### Context

To override dependabot PRs.

### CODE REVIEW

File: tee-worker/service/src/main_impl.rs [hunk 1]
```diff
@@ -241,9 +241,9 @@ pub(crate) fn main() {
 	} else if matches.is_present("shielding-key") {
 		setup::generate_shielding_key_file(enclave.as_ref());
 	} else if matches.is_present("signing-key") {
-		setup::generate_signing_key_file(enclave.as_ref());
 		let tee_accountid = enclave_account(enclave.as_ref());
-		println!("Enclave signing account: {:}", &tee_accountid.to_ss58check());
+		println!("Enclave account: {:}", &tee_accountid.to_ss58check());
+		setup::generate_signing_key_file(enclave.as_ref());
```
Comment (kziemianek) [inline tee-worker/service/src/main_impl.rs]: Is this change intentional? 🤔

Comment (Kailai-Wang) [reply tee-worker/service/src/main_impl.rs]: No, sorry. It must come from the merging.

I reverted it and adjusted `bitacross-worker/srvice/src/main_impl.rs`

**Review by kziemianek — APPROVED** [review]
